metrics/internal/collectors/registry.go

```diff
@@ -29,3 +41,81 @@ func RegisterCustomResourceCollectors(registry *prometheus.Registry, opts *optio
 		OBMetricsCollector,
 	)
 }
+
+var pvStoreEnabled bool
+var pvStore = internalcache.NewPersistentVolumeStore()
+
+func enablePVStore(opts *options.Options) {
+	client := clientset.NewForConfigOrDie(opts.Kubeconfig)
+	lw := internalcache.CreatePersistentVolumeListWatch(client, "")
+	reflector := cache.NewReflector(lw, &corev1.PersistentVolume{}, pvStore, 0)
+	go reflector.Run(opts.StopCh)
+	pvStoreEnabled = true
+}
+
+var rbdMirrorStoreEnabled bool
+var rbdMirrorStore = internalcache.NewRBDMirrorStore()
+
+type CSIClusterConfig struct {
+	ClusterID string   `json:"clusterID"`
+	Monitors  []string `json:"monitors"`
+}
+
+func enableRBDMirrorStore(opts *options.Options) error {
+	client := clientset.NewForConfigOrDie(opts.Kubeconfig)
+	secret, err := client.CoreV1().Secrets("rook-ceph").Get(context.TODO(), "rook-ceph-mon", v1.GetOptions{})
```

Comment:
replace rook-ceph with openshit-storage in all places

Reply:
Will fetch it from env variable, so that we don't hardcode namespace.

Reply:
using hardcoded namespace for now since env var is not easily available.